gamd/config.py

```diff
@@ -198,13 +197,13 @@ def __init__(self):
         self.restart_checkpoint_interval = 50000
         self.statistics_interval = 500
         return
-    
+
```

Comment:
There are a lot of spacing changes unrelated to the functionality change that is being made.  For future reference, this makes it much more difficult to review the change, and it would be preferable to have this kind of change in it's own pull request.